src/base_guicontrol.cpp

```diff
@@ -109,3 +109,13 @@ std::unordered_map<std::string, double> MainFrame::generateRadiusMapFromView(){
   }
   return radius_map;
 }
+
+std::vector<std::string> MainFrame::getIncludedElementsFromView(){

```

Comment:
"FromView" should be clear from the fact that this is a function of the GUI class.

Reply:
I agree but I simply followed your naming convention (see the function right above: `generateRadiusMapFromView` ).
Wasn't it your doing?

Reply:
I think that was originally one of your functions as well, but I may have added the "FromView". I don't remember. But in both cases, I think having such wordy function names, makes the code more difficult to read. I just added this here, since the function is new, and changing its name is still simple enough.

Reply:
Ok, I'll remove both "FromView". I suppose that your original intention for adding "FromView" was to differentiate radius map generated from file and from grid but as you wrote, the GUI class is self-explanatory.